db/migrate/20170918173132_create_tournaments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTournaments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tournaments do |t|
+      t.string :name
```

Comment:
Consider using `null: false` for columns that require data.

Reply:
fixed